src/core-services/account/schemas/account.graphql

```diff
@@ -136,6 +136,19 @@ input RemoveAccountEmailRecordInput {
   email: Email!
 }
 
+"Defines groups to add given user / account to thereby giving that user certain permissions"
+input SetUserPermissionsInput {
+
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
```

Comment:
This allows `clientMutationId` but nothing is done with it. It is in the response schema but never returned.

Reply:
`clientMutationId` is an `echo`, so it needs to be returned in the mutation resolver as is, along with the other results from the mutation.

Reply:
@mikemurray @rosshadden ... done